README.md

```diff
@@ -173,15 +173,17 @@ Example:
 <!-- AUTHORS -->
 
 ## 👥 Authors <a name="authors">Syed</a>
+# <a name="authors">Syed Atril</a>
+👤 **Syed Atril**
+
 
 > This Project is created by Syed Atril
 
 ## 🛠 Built With <a name="built-with"></a>
 - HTML
 - CSS
 - JAVASCRIPT
-# <a name="authors">Syed Atril</a>
-👤 **Syed Atril**
+
 
 - GitHub: [@githubhandle](https://github.com/Atril33)
 - Twitter: [@twitterhandle](https://twitter.com/AtrilSyed)
```

Comment:
- [OPTIONAL] I have observed that you have misplaced the name of the author in your README file. Kindly relocate it to its original position and remember to include the details about your partner :+1: